client/src/components/DataFiles/DataFilesStatus/DataFilesStatus.js

```diff
@@ -0,0 +1,60 @@
+import { findSystemDisplayName } from 'utils/systems';
+import truncateMiddle from '../../../utils/truncateMiddle';
+
+const OPERATION_MAP = {
```

Comment:
Some cases are combined (`upload`, `move`, `copy`), and some operations have the same action verb (`move` and `trash` are both `moved`), making the mapping a bit more useful than convoluted imo. Though I agree, it is a bit overkill. But for now I wanted to keep a similar mechanism to [how job notifications are translated](https://github.com/TACC/Frontera-Portal/blob/master/client/src/components/Jobs/JobsStatus/JobsStatus.js#L9)

Perhaps this can be refactored in a follow up ticket

Reply:
I weigh the benefit of legibility and ease of string editing as greater than the benefit of "DRY-er" string content. 

> I support consistency, but [how job notifications are translated](https://github.com/TACC/Frontera-Portal/blob/master/client/src/components/Jobs/JobsStatus/JobsStatus.js#L9) is literally a mapping of API verbiage to standalone user string. This instance here leans more toward language construction logic, than API-to-User translation.
>
> The toast texts are essentially translation from dev process to user language. And, this code assumes complex responsibility that belongs to the UI designer. If a dev is asked to add or change a string, the task could be an inconsequential one-line change; but with this code, that task may require a change to logic.